src/Compiler/Interactive/fsi.fs

```diff
@@ -4763,6 +4768,7 @@ type FsiEvaluationSession
         let userRes =
             match res with
             | Choice1Of2 r -> Choice1Of2 r
+            | Choice2Of2 None when errorInfos.Length = 0 && tcConfigB.typeCheckOnly -> Choice1Of2 None
```

Comment:
Because of this change, line numbers after ilxgen for closures have shifted and therefore slightly changed ilverify baseline.

Read this repo docs for `ilverify` (search for it) and follow the guide for updating the baseline. It will require setting an env var and then running a script.